scripts/jenkins/ci.suse.de/openstack-mkcloud-pr-trigger.xml

```diff
@@ -65,32 +60,39 @@ case $mode in
   ;;
 esac
 
-# PR testing is only possible on SLE12 nodes, so set a label
-staticconfig="label=openstack-mkcloud-SLE12"
-
-## mkcloud self-gating
 ghs=${automationrepo}/scripts/github-status/github-status.rb
 
-prs=${automation_prs}
-prs=${prs:-`$ghs -a ${action}`}
-
-for automation_pr in $prs ; do
-  automation_opts=(${automation_pr//:/ })
-  automation_pr_sha=${automation_opts[1]}
+declare -a cloud_repos=(
+    SUSE-Cloud/automation
+    SUSE-Cloud/cct
+)
 
-  # default config / may be overridden by tags from PR message
-  mkcloudtarget=all_noreboot
-  cloudsource=develcloud5
-  nodenumber=2
-  networkingplugin=openvswitch
-  #echo eval `$ghs -a get-pr-config -p $automation_pr_sha` # not yet implemented
-  ${automationrepo}/scripts/jenkins/jenkins-job-trigger openstack-mkcloud -p mode=standard github_pr=SUSE-Cloud/automation:${automation_pr} mkcloudtarget=${mkcloudtarget} cloudsource="${cloudsource}" nodenumber=${nodenumber} networkingplugin=${networkingplugin} $staticconfig
-  $ghs -a set-status -s "pending" -t $BUILD_URL -c $automation_pr_sha
+for repo in ${cloud_repos[@]}; do
+    for github_pr in $($ghs -r $repo -a $action); do
+        github_pr_opts=(${github_pr//:/ })
+        github_pr_sha=${github_pr_opts[1]}
+        github_pr_branch=${github_pr_opts[2]}
+
+        if [ "$github_pr_branch" = "master" ]; then
+            # default config / may be overridden by tags from PR message
+            mkcloudtarget=all_noreboot
+            cloudsource=develcloud5
```

Comment:
I don't know why we decided to pick develcloud5 (also without TESTHEAD=1 set). but this is just code reindentation, I didn't want to change this in this commit..